pelican/tests/build_test/test_wheel.py

```diff
@@ -0,0 +1,28 @@
+from pathlib import Path
+import pytest
+from zipfile import ZipFile
+
+
+@pytest.mark.skipif(
+    "not config.getoption('--check-wheel')",
+    reason="Only run when --check-wheel is given",
+)
+def test_wheel_contents(pytestconfig):
+    """
+    This test, should test the contents of the wheel to make sure,
+    that everything that is needed is included in the final build
+    """
+    wheel_file = pytestconfig.getoption("--check-wheel")
+    assert wheel_file.endswith(".whl")
+    files_list = ZipFile(wheel_file).namelist()
+    ## Check is theme files are copiedto wheel
```

Comment:
Maybe a single `#` instead of two? Also, there are some errors. I would change the text to:

    # Check if theme files are copied to wheel